yarn-project/blob-sink/src/blobstore/blob_store_test_suite.ts

```diff
@@ -0,0 +1,94 @@
+import { Blob } from '@aztec/foundation/blob';
+import { Fr } from '@aztec/foundation/fields';
+
+import { BlobWithIndex } from '../types/index.js';
+import { type BlobStore } from './interface.js';
+
+export function describeBlobStore(getBlobStore: () => BlobStore) {
+  let blobStore: BlobStore;
+
+  beforeEach(() => {
+    blobStore = getBlobStore();
+  });
+
+  it('should store and retrieve a blob', async () => {
+    // Create a test blob with random fields
+    const testFields = [Fr.random(), Fr.random(), Fr.random()];
+    const blob = Blob.fromFields(testFields);
+    const blockId = '12345';
```

Comment:
I notice id here is string with numbers but earlier we had hex. Would be good to check that they behave as expected, e.g., `0x1234` and `4660` is the same thing.